modules/src/hello.py

```diff
@@ -13,13 +13,20 @@ def process(input, entities=None):
         'Oh hello, sir!',
         'Perhaps, if you intend to visit other planets, we should improve the exosystems.',
     ]
+
+    respone = random.choice(greetings)
+
+    personalized = False
     if entities is not None:
         if 'sender' in entities and 'first_name' in entities['sender']:
+            personalized = True
             sender_name = entities['sender']['first_name']
-            greetings = [greeting.replace('sir', sender_name) for greeting in greetings]
```

Comment:
I meant we can just change this single line to have the replacement case-insensitive. 😄 
What do you think?

Reply:
`re`!